### PR TITLE
Initialzation for school

### PR DESCRIPTION
**Pull request by robertguturing**
Creating school class

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class School
+  attr_accessor :start_time, :hours_in_day
```
Comment (Pragmaticpraxis37) [inline lib/school.rb]: Hey have you thought about using this as two lines?

Comment (Pragmaticpraxis37) [reply lib/school.rb]: Have you thought about using an attr_reader?  We don't want people to change the attribute outside of the class.

**Review by Pragmaticpraxis37 — CHANGES_REQUESTED** [review]
Hey, could you add some tests as well?

File: lib/school.rb (hunk 1 above)
Comment (robertguturing) [reply lib/school.rb]: Thanks for catching those! I'll make the changes right away 🖖

**Review by Pragmaticpraxis37 — APPROVED** [review]

**Comment by Pragmaticpraxis37** [comment]
Ship it.

**Pull request merged**